Tests/TuistCacheTests/ContentHashing/CoreDataModelsContentHasherTests.swift

```diff
@@ -0,0 +1,91 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+
+final class CoreDataModelsContentHasherTests: TuistUnitTestCase {
+    private var sut: CoreDataModelsContentHasher!
+    private var coreDataModel: CoreDataModel!
+    private var temporaryDirectory: TemporaryDirectory!
+    private var mockContentHasher: MockContentHashing!
+    private let defaultValuesHash = "05c9d517e2cf12b45786787dae929a23" // Expected hash for the CoreDataModel created with the buildCoreDataModel function using default values
+    override func setUp() {
+        super.setUp()
+        mockContentHasher = MockContentHashing()
+        sut = CoreDataModelsContentHasher(contentHasher: mockContentHasher)
+        do {
+            temporaryDirectory = try TemporaryDirectory(removeTreeOnDeinit: true)
+        } catch {
+            XCTFail("Error while creating temporary directory")
+        }
+    }
+
+    override func tearDown() {
+        sut = nil
+        coreDataModel = nil
+        temporaryDirectory = nil
+        mockContentHasher = nil
+        super.tearDown()
+    }
+
+    // MARK: - Tests
+
+    func test_hash_returnsSameValue() throws {
+        coreDataModel = try buildCoreDataModel()
+        mockContentHasher.hashStringsStub = "fixed"
```

Comment:
Notice that with the way this test is written, we are saying: regardless of the string that we are trying to hash, always return `fixed`. I'd change the mock a bit so that we return a different hash depending on the input. Indeed, we can say for every string `x`, the returned hash will be the same string prefixed by `hashed-`.

Reply:
Smart! Make sense, I've updated all tests with this suggestion